src/agent/configuration_parser/src/configuration_parser.cpp

```diff
@@ -19,10 +19,25 @@ namespace configuration
             LogError("Using default values due to error parsing wazuh.conf file: {}", e.what());
 
             tbl = toml::parse_str(
-                R"([agent]
+                R"(
+                [agent]
                 server_mgmt_api_port = "55000"
                 agent_comms_api_port = "27000"
-                manager_ip = "localhost")",
+                manager_ip = "localhost"
+
+                [inventory]
+                disabled = false
```

Comment:
Maybe it's better to name this setting `enabled`.

Reply:
done alongside tests and several functions.